tools/clang/test/CodeGenDXIL/hlsl/shader-flags/tiled_res_sample.hlsl

```diff
@@ -0,0 +1,19 @@
+// RUN: %dxc -T ps_6_0 %s | FileCheck %s
```

Comment:
Apparently, there was an update that already covered some of the feedback I had in my initial comment on the tests.

Revising:

Related to my other comment, I think new cases need to be tested: make sure that both no clamp usage and using constant `0.0f` for clamp does not set the flag.  Also, if a constant value `0.0f` is tested to make sure the flag doesn't get set, then we should probably have another non-zero constant value to make sure it does get set in that case.

I think all these variants could be implemented in one test with multiple RUN lines rather than a bunch of separate test files with duplicated run lines.  I don't know what others prefer, but IMO if you have a test that only varies something minor that could be varied in a run line, it's a lot easier to tell that's the case, see the breadth of coverage, and maintain the tests if written in a more templated way using multiple run lines.